manifest.yml

```diff
@@ -3,6 +3,8 @@
 memory: 1G
 instances: 1
 applications:
-- name: foia
+- name: krang
   path: .
   timeout: 180
+  services:
+  - es
```

Comment:
Is this enough to start an es service? Does this bind the service to the applicatoin?